e2e/gyp_no_install_script/MODULE.bazel

```diff
@@ -8,6 +9,9 @@ npm = use_extension("@aspect_rules_js//npm:extensions.bzl", "npm")
 npm.npm_translate_lock(
     name = "npm",
     data = ["//:package.json"],
+    lifecycle_hooks_use_default_shell_env = {
+        "segfault-handler": "true",
```

Comment:
Have we had to do this elsewhere where we want a dict of boolean values? In WORKSPACE it is a macro and the use of strings is hidden unlike here...

Reply:
a dict of boolean values seems like it ought to just be a list? I don't think the datatype has to match the other lifecycle_hooks_* necessarily

Reply:
this will be rarely used so I'm fine with the option that makes maintenance easier rather than the more ergonomic one.